src/ppom-controller.ts

```diff
@@ -337,25 +347,41 @@ export class PPOMController extends BaseControllerV2<
    * This function receives a callback that will be called with the PPOM.
    *
    * @param callback - Callback to be invoked with PPOM.
+   * @param networkClientId - Optional network client ID to use for the PPOM requests.
    */
   async usePPOM<T>(
     callback: (ppom: any) => Promise<T>,
```

Comment:
same as above https://github.com/MetaMask/ppom-validator/pull/85/files#r1387128172
```suggestion
    callback: (ppom: unknown) => Promise<T>,
```